INSTALL.md

```diff
@@ -195,3 +200,69 @@ For example, in a Linux platform, assuming one wishes to execute the `adaboost_d
 ```shell
 ./_cmake_results/intel_intel64_so/adaboost_dense_batch
 ```
+
+## Installation with a conda environment
+
+The previous instructions assumed system-wide installs of the necessary dependencies. These can also be installed at a user-level through the `conda` or [mamba](https://github.com/conda-forge/miniforge) ecosystems.
+
+To create a conda environment for building oneDAL, after `conda` has been installed:
+
+```shell
+conda create -y -n onedal_env
+conda activate onedal_env
+```
+
+To install the necessary dependencies with `conda`, **assuming a linux system**:
+
+```shell
+conda install -y \
+    -c https://software.repos.intel.com/python/conda/ \ `# Intel's repository`
+    -c conda-forge \ `# conda-forge, for tools like 'make'`
+    make python \ `# used by the build system`
+    dpcpp-cpp-rt dpcpp_linux-64 intel-sycl-rt \ `# Intel compiler packages`
+    tbb tbb-devel \ `# required TBB packages`
+    mkl mkl-devel mkl-static mkl-dpcpp mkl-devel-dpcpp \ `# required MKL packages`
+    cmake `# required to build the examples only`
+```
+
+_(for windows, replace `dpcpp_linux-64` with `dpcpp_win-64`, and remove the comments within backticks)_
```

Comment:
Let's have separate instructions for Linux and Windows - save the time for users on copy-pasting.

Also, i think the minimal supported versions for the packages (at least some) need to be added. Like python=3.9, etc.

Reply:
Changed to separate code snippets.